### PR TITLE
andrew week1

### PR DESCRIPTION
**Pull request by ddvkid**

### CODE REVIEW

File: cis194/week1/andrew/Homework.hs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+module Homework
+    ( toDigits,
+      toDigitsRev
```
Comment (cht8687) [inline cis194/week1/andrew/Homework.hs]: check this best practise:
https://github.com/fp-works/2019-winter-Haskell-school/pull/7#discussion_r312690736

File: cis194/week1/andrew/Homework.hs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+type Disks = Integer
+type Peg   = String
+type Move  = (Peg, Peg)
+
+hanoi :: Disks -> Peg -> Peg -> Peg -> [Move]
+
+hanoi n a b c
+  | n > 0 = hanoi (n-1) a c b 
+            ++ [(a,b)]
+            ++ hanoi (n-1) c b a
```
Comment (cht8687) [inline cis194/week1/andrew/Homework.hs]: check this best practice in terms of indentation: https://github.com/fp-works/2019-winter-Haskell-school/pull/4#discussion_r312450284

**Comment by cht8687** [comment]
well done. 👍

File: cis194/week1/andrew/Homework.hs (hunk 1 above)
Comment (ddvkid) [reply cis194/week1/andrew/Homework.hs]: sure will do.

**Pull request merged**